LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 Calliope
+Copyright (c) 2024 AUTHORS
```

Comment:
In calliope we do `contributors listed in AUTHORS.` to make it clearer. Don't mind either way.

Reply:
I'll leave this as-is for now. Later we might move to a CITATION and LICENSE method, depending on what the expert says.